### PR TITLE
Dell AT101 replacement pcb support

### PR DESCRIPTION
**Pull request by henrikosorensen**
Add support for GPL2'ed AT101 and family replacement PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/at101ish/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Keyboard Maintainer: QMK Community and Henrik O. Sørensen  
+Hardware Supported: AT10[12]W? PCB  
+Hardware Availability: https://github.com/henrikosorensen/keyboard_pcbs/tree/master/at101_pcb
+
+Make example for this keyboard (after setting up your build environment):
+
+    make at101ish:default
+
+See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```
Comment (fauxpark) [inline keyboards/at101ish/readme.md]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/at101ish/rules.mk [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# keyboard specific files
```
Comment (fauxpark) [inline keyboards/at101ish/rules.mk]:
```suggestion
```

File: keyboards/at101ish/rules.mk [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+SRC +=
```
Comment (fauxpark) [inline keyboards/at101ish/rules.mk]:
```suggestion
```

File: keyboards/at101ish/rules.mk [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# keyboard specific files
+SRC += 
+
```
Comment (fauxpark) [inline keyboards/at101ish/rules.mk]:
```suggestion
```

**Comment by henrikosorensen** [comment]
> Could you move this to the "handwired" directory?
> 
> Specifically, this looks like it's a passion project, and not mass produced in any way (eg, not sold). In this case, the handwired directory is where we tend to put these type of keyboards.

If that what you deem best? But there has to 100000s of Dell AT101 series boards out there needing a QMK upgraded PCB? And this one is available freely, not in a once in a lifetime group buy.... but I'll move it if you still think it's best. :)

**Comment by drashna** [comment]
Sorry for the delay. 

And yes.  "needing" and "having" are two different things.   Basically, because this is a custom board, that isn't being mass produced, from the looks of it, we'd rather keep it in the handwired directory as that's the location for a lot of "passion projects" like this.

**Comment by noroadsleft** [comment]
Regarding file location, I'm actually okay with this being directly in `keyboards/at101ish/`. The Omnikey(ish) is directly in `keyboards/` as well, and has the same availability ([open sourced on GitHub](https://github.com/henrikosorensen/keyboard_pcbs/tree/master/at101_pcb)).

File: keyboards/at101ish/readme.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+AT101(-ish)
```
Comment (noroadsleft) [inline keyboards/at101ish/readme.md]: Some markdown consistency (with QMK at large) would be nice.

```suggestion
# AT101(-ish)
```

File: keyboards/at101ish/readme.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+===
```
Comment (noroadsleft) [inline keyboards/at101ish/readme.md]:
```suggestion
```

File: keyboards/at101ish/readme.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+A replacement PCB for Dell AT101 family. (In theory) supports 101, 101W, 102, 102W, maybe other bigfoot platform boards.
+
+Keyboard Maintainer: QMK Community and Henrik O. Sørensen  
+Hardware Supported: AT10[12]W? PCB
```
Comment (noroadsleft) [inline keyboards/at101ish/readme.md]:
```suggestion
Hardware Supported: AT101(ish) PCB  
```

File: keyboards/at101ish/info.json [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+                       {"label":"Tab", "x":0, "y":2.5, "w":1.5}, {"label":"Q", "x":1.5, "y":2.5}, {"label":"W", "x":2.5, "y":2.5}, {"label":"E", "x":3.5, "y":2.5}, {"label":"R", "x":4.5, "y":2.5}, {"label":"T", "x":5.5, "y":2.5}, {"label":"Y", "x":6.5, "y":2.5}, {"label":"U", "x":7.5, "y":2.5}, {"label":"I", "x":8.5, "y":2.5}, {"label":"O", "x":9.5, "y":2.5}, {"label":"P", "x":10.5, "y":2.5}, {"label":"{", "x":11.5, "y":2.5}, {"label":"}", "x":12.5, "y":2.5}, {"label":"Enter", "x":13.75, "y":2.5, "w":1.25, "h":2}, {"label":"Delete", "x":15.25, "y":2.5}, {"label":"End", "x":16.25, "y":2.5}, {"label":"PgDn", "x":17.25, "y":2.5}, {"label":"7", "x":18.5, "y":2.5}, {"label":"8", "x":19.5, "y":2.5}, {"label":"9", "x":20.5, "y":2.5}, {"label":"+", "x":21.5, "y":2.5, "h":2},
```
Comment (noroadsleft) [inline keyboards/at101ish/info.json]: Enter's object is out of place for the AT102 and 102W layouts.

```suggestion
                       {"label":"Tab", "x":0, "y":2.5, "w":1.5}, {"label":"Q", "x":1.5, "y":2.5}, {"label":"W", "x":2.5, "y":2.5}, {"label":"E", "x":3.5, "y":2.5}, {"label":"R", "x":4.5, "y":2.5}, {"label":"T", "x":5.5, "y":2.5}, {"label":"Y", "x":6.5, "y":2.5}, {"label":"U", "x":7.5, "y":2.5}, {"label":"I", "x":8.5, "y":2.5}, {"label":"O", "x":9.5, "y":2.5}, {"label":"P", "x":10.5, "y":2.5}, {"label":"{", "x":11.5, "y":2.5}, {"label":"}", "x":12.5, "y":2.5}, {"label":"Delete", "x":15.25, "y":2.5}, {"label":"End", "x":16.25, "y":2.5}, {"label":"PgDn", "x":17.25, "y":2.5}, {"label":"7", "x":18.5, "y":2.5}, {"label":"8", "x":19.5, "y":2.5}, {"label":"9", "x":20.5, "y":2.5}, {"label":"+", "x":21.5, "y":2.5, "h":2},
```

File: keyboards/at101ish/info.json [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+                       {"label":"Caps Lock", "x":0, "y":3.5, "w":1.75}, {"label":"A", "x":1.75, "y":3.5}, {"label":"S", "x":2.75, "y":3.5}, {"label":"D", "x":3.75, "y":3.5}, {"label":"F", "x":4.75, "y":3.5}, {"label":"G", "x":5.75, "y":3.5}, {"label":"H", "x":6.75, "y":3.5}, {"label":"J", "x":7.75, "y":3.5}, {"label":"K", "x":8.75, "y":3.5}, {"label":"L", "x":9.75, "y":3.5}, {"label":":", "x":10.75, "y":3.5}, {"label":"@", "x":11.75, "y":3.5}, {"label":"~", "x":12.75, "y":3.5}, {"label":"4", "x":18.5, "y":3.5}, {"label":"5", "x":19.5, "y":3.5}, {"label":"6", "x":20.5, "y":3.5},
```
Comment (noroadsleft) [inline keyboards/at101ish/info.json]:
```suggestion
                       {"label":"Caps Lock", "x":0, "y":3.5, "w":1.75}, {"label":"A", "x":1.75, "y":3.5}, {"label":"S", "x":2.75, "y":3.5}, {"label":"D", "x":3.75, "y":3.5}, {"label":"F", "x":4.75, "y":3.5}, {"label":"G", "x":5.75, "y":3.5}, {"label":"H", "x":6.75, "y":3.5}, {"label":"J", "x":7.75, "y":3.5}, {"label":"K", "x":8.75, "y":3.5}, {"label":"L", "x":9.75, "y":3.5}, {"label":":", "x":10.75, "y":3.5}, {"label":"@", "x":11.75, "y":3.5}, {"label":"~", "x":12.75, "y":3.5}, {"label":"Enter", "x":13.75, "y":2.5, "w":1.25, "h":2}, {"label":"4", "x":18.5, "y":3.5}, {"label":"5", "x":19.5, "y":3.5}, {"label":"6", "x":20.5, "y":3.5},
```

File: keyboards/at101ish/info.json [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+                       {"label":"Tab", "x":0, "y":2.5, "w":1.5}, {"label":"Q", "x":1.5, "y":2.5}, {"label":"W", "x":2.5, "y":2.5}, {"label":"E", "x":3.5, "y":2.5}, {"label":"R", "x":4.5, "y":2.5}, {"label":"T", "x":5.5, "y":2.5}, {"label":"Y", "x":6.5, "y":2.5}, {"label":"U", "x":7.5, "y":2.5}, {"label":"I", "x":8.5, "y":2.5}, {"label":"O", "x":9.5, "y":2.5}, {"label":"P", "x":10.5, "y":2.5}, {"label":"{", "x":11.5, "y":2.5}, {"label":"}", "x":12.5, "y":2.5}, {"label":"Enter", "x":13.75, "y":2.5, "w":1.25, "h":2}, {"label":"Delete", "x":15.25, "y":2.5}, {"label":"End", "x":16.25, "y":2.5}, {"label":"PgDn", "x":17.25, "y":2.5}, {"label":"7", "x":18.5, "y":2.5}, {"label":"8", "x":19.5, "y":2.5}, {"label":"9", "x":20.5, "y":2.5}, {"label":"+", "x":21.5, "y":2.5, "h":2},
```
Comment (noroadsleft) [inline keyboards/at101ish/info.json]:
```suggestion
                       {"label":"Tab", "x":0, "y":2.5, "w":1.5}, {"label":"Q", "x":1.5, "y":2.5}, {"label":"W", "x":2.5, "y":2.5}, {"label":"E", "x":3.5, "y":2.5}, {"label":"R", "x":4.5, "y":2.5}, {"label":"T", "x":5.5, "y":2.5}, {"label":"Y", "x":6.5, "y":2.5}, {"label":"U", "x":7.5, "y":2.5}, {"label":"I", "x":8.5, "y":2.5}, {"label":"O", "x":9.5, "y":2.5}, {"label":"P", "x":10.5, "y":2.5}, {"label":"{", "x":11.5, "y":2.5}, {"label":"}", "x":12.5, "y":2.5}, {"label":"Delete", "x":15.25, "y":2.5}, {"label":"End", "x":16.25, "y":2.5}, {"label":"PgDn", "x":17.25, "y":2.5}, {"label":"7", "x":18.5, "y":2.5}, {"label":"8", "x":19.5, "y":2.5}, {"label":"9", "x":20.5, "y":2.5}, {"label":"+", "x":21.5, "y":2.5, "h":2},
```

File: keyboards/at101ish/info.json [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+                       {"label":"Caps Lock", "x":0, "y":3.5, "w":1.75}, {"label":"A", "x":1.75, "y":3.5}, {"label":"S", "x":2.75, "y":3.5}, {"label":"D", "x":3.75, "y":3.5}, {"label":"F", "x":4.75, "y":3.5}, {"label":"G", "x":5.75, "y":3.5}, {"label":"H", "x":6.75, "y":3.5}, {"label":"J", "x":7.75, "y":3.5}, {"label":"K", "x":8.75, "y":3.5}, {"label":"L", "x":9.75, "y":3.5}, {"label":":", "x":10.75, "y":3.5}, {"label":"@", "x":11.75, "y":3.5}, {"label":"~", "x":12.75, "y":3.5}, {"label":"4", "x":18.5, "y":3.5}, {"label":"5", "x":19.5, "y":3.5}, {"label":"6", "x":20.5, "y":3.5},
```
Comment (noroadsleft) [inline keyboards/at101ish/info.json]:
```suggestion
                       {"label":"Caps Lock", "x":0, "y":3.5, "w":1.75}, {"label":"A", "x":1.75, "y":3.5}, {"label":"S", "x":2.75, "y":3.5}, {"label":"D", "x":3.75, "y":3.5}, {"label":"F", "x":4.75, "y":3.5}, {"label":"G", "x":5.75, "y":3.5}, {"label":"H", "x":6.75, "y":3.5}, {"label":"J", "x":7.75, "y":3.5}, {"label":"K", "x":8.75, "y":3.5}, {"label":"L", "x":9.75, "y":3.5}, {"label":":", "x":10.75, "y":3.5}, {"label":"@", "x":11.75, "y":3.5}, {"label":"~", "x":12.75, "y":3.5}, {"label":"Enter", "x":13.75, "y":2.5, "w":1.25, "h":2}, {"label":"4", "x":18.5, "y":3.5}, {"label":"5", "x":19.5, "y":3.5}, {"label":"6", "x":20.5, "y":3.5},
```

File: keyboards/at101ish/at101ish.h [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+	{ K0511, K0512, K0513, K0514, K0515, _____, K0516, K0517, K0518, K0519 }, \
+	{ K0601, K0602, K0603, _____, _____, _____, _____, K0604, _____, _____ }, \
+	{ K0605, K0606, K0607, K0608, K0609, K0610, K0611, K0612, K0613, _____ }  \
+}
+
+#define LAYOUT_at101( \
+	K0101,        K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110, K0111, K0112, K0113,   K0114, K0115, K0116, \
+	K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210, K0211, K0212, K0213, K0214,   K0215, K0216, K0217,   K0218, K0219, K0220, K0221, \
+	K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310, K0311, K0312, K0313, K0314,   K0315, K0316, K0317,   K0318, K0319, K0320, K0321, \
+	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412,    K0414,                              K0415, K0416, K0417, K0418, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]: Only `LAYOUT_all` would use `K0418`.

```suggestion
	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412,    K0414,                              K0415, K0416, K0417,        \
```

File: keyboards/at101ish/at101ish.h [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+	K0601,        K0603,                         K0604,                   K0605,               K0608,   K0609, K0610, K0611,   K0612,        K0613         \
+) { \
+	{ K0101, K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110 }, \
+	{ K0111, K0112, K0113, K0114, K0115, K0116, _____, _____, K0321, K0221 }, \
+	{ K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210 }, \
+	{ K0211, K0212, K0213, K0214, K0215, K0216, K0217, K0218, K0219, K0220 }, \
+	{ K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310 }, \
+	{ K0311, K0312, K0313, K0314, K0315, K0316, K0317, K0318, K0319, K0320 }, \
+	{ K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410 }, \
+	{ K0411, K0412, _____, K0414, _____, _____, K0415, K0416, K0417, K0418 }, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]:
```suggestion
	{ K0411, K0412, _____, K0414, _____, _____, K0415, K0416, K0417, _____ }, \
```

File: keyboards/at101ish/at101ish.h [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+	{ K0511, K0512, K0513, _____, K0515, _____, K0516, K0517, K0518, K0519 }, \
+	{ K0601, _____, K0603, _____, _____, _____, _____, K0604, _____, _____ }, \
+	{ K0605, _____, _____, K0608, K0609, K0610, K0611, K0612, K0613, _____ }  \
+}
+
+#define LAYOUT_at101w( \
+	K0101,        K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110, K0111, K0112, K0113,   K0114, K0115, K0116, \
+	K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210, K0211, K0212, K0213, K0214,   K0215, K0216, K0217,   K0218, K0219, K0220, K0221, \
+	K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310, K0311, K0312, K0313, K0314,   K0315, K0316, K0317,   K0318, K0319, K0320, K0321, \
+	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412,    K0414,                              K0415, K0416, K0417, K0418, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]:
```suggestion
	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412,    K0414,                              K0415, K0416, K0417,        \
```

File: keyboards/at101ish/at101ish.h [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+	K0601, K0602, K0603,                         K0604,                   K0605, K0606, K0607, K0608,   K0609, K0610, K0611,   K0612,        K0613         \
+) { \
+	{ K0101, K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110 }, \
+	{ K0111, K0112, K0113, K0114, K0115, K0116, _____, _____, K0321, K0221 }, \
+	{ K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210 }, \
+	{ K0211, K0212, K0213, K0214, K0215, K0216, K0217, K0218, K0219, K0220 }, \
+	{ K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310 }, \
+	{ K0311, K0312, K0313, K0314, K0315, K0316, K0317, K0318, K0319, K0320 }, \
+	{ K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410 }, \
+	{ K0411, K0412, _____, K0414, _____, _____, K0415, K0416, K0417, K0418 }, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]:
```suggestion
	{ K0411, K0412, _____, K0414, _____, _____, K0415, K0416, K0417, _____ }, \
```

File: keyboards/at101ish/at101ish.h [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+	{ K0511, K0512, K0513, _____, K0515, _____, K0516, K0517, K0518, K0519 }, \
+	{ K0601, K0602, K0603, _____, _____, _____, _____, K0604, _____, _____ }, \
+	{ K0605, K0606, K0607, K0608, K0609, K0610, K0611, K0612, K0613, _____ }  \
+}
+
+#define LAYOUT_at102( \
+	K0101,        K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110, K0111, K0112, K0113,   K0114, K0115, K0116, \
+	K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210, K0211, K0212, K0213, K0214,   K0215, K0216, K0217,   K0218, K0219, K0220, K0221, \
+	K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310, K0311, K0312, K0313,          K0315, K0316, K0317,   K0318, K0319, K0320, K0321, \
+	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412, K0413, K0414,                          K0415, K0416, K0417, K0418, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]:
```suggestion
	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412, K0413, K0414,                          K0415, K0416, K0417,        \
```

File: keyboards/at101ish/at101ish.h [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+	K0601,        K0603,                      K0604,                      K0605,               K0608,   K0609, K0610, K0611,   K0612,        K0613         \
+) { \
+	{ K0101, K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110 }, \
+	{ K0111, K0112, K0113, K0114, K0115, K0116, _____, _____, K0321, K0221 }, \
+	{ K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210 }, \
+	{ K0211, K0212, K0213, K0214, K0215, K0216, K0217, K0218, K0219, K0220 }, \
+	{ K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310 }, \
+	{ K0311, K0312, K0313, _____, K0315, K0316, K0317, K0318, K0319, K0320 }, \
+	{ K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410 }, \
+	{ K0411, K0412, K0413, K0414, _____, _____, K0415, K0416, K0417, K0418 }, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]:
```suggestion
	{ K0411, K0412, K0413, K0414, _____, _____, K0415, K0416, K0417, _____ }, \
```

File: keyboards/at101ish/at101ish.h [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+	{ K0511, K0512, K0513, _____, K0515, _____, K0516, K0517, K0518, K0519 }, \
+	{ K0601, _____, K0603, _____, _____, _____, _____, K0604, _____, _____ }, \
+	{ K0605, _____, _____, K0608, K0609, K0610, K0611, K0612, K0613, _____ }  \
+}
+
+#define LAYOUT_at102w( \
+	K0101,        K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110, K0111, K0112, K0113,   K0114, K0115, K0116, \
+	K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210, K0211, K0212, K0213, K0214,   K0215, K0216, K0217,   K0218, K0219, K0220, K0221, \
+	K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310, K0311, K0312, K0313,          K0315, K0316, K0317,   K0318, K0319, K0320, K0321, \
+	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412, K0413, K0414,                          K0415, K0416, K0417, K0418, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]:
```suggestion
	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412, K0413, K0414,                          K0415, K0416, K0417,        \
```

File: keyboards/at101ish/at101ish.h [hunk 19]
```diff
@@ -0,0 +1,116 @@
...
+	K0601, K0602, K0603,                      K0604,                      K0605, K0606, K0607, K0608,   K0609, K0610, K0611,   K0612,        K0613         \
+) { \
+	{ K0101, K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110 }, \
+	{ K0111, K0112, K0113, K0114, K0115, K0116, _____, _____, K0321, K0221 }, \
+	{ K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210 }, \
+	{ K0211, K0212, K0213, K0214, K0215, K0216, K0217, K0218, K0219, K0220 }, \
+	{ K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310 }, \
+	{ K0311, K0312, K0313, _____, K0315, K0316, K0317, K0318, K0319, K0320 }, \
+	{ K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410 }, \
+	{ K0411, K0412, K0413, K0414, _____, _____, K0415, K0416, K0417, K0418 }, \
```
Comment (noroadsleft) [inline keyboards/at101ish/at101ish.h]:
```suggestion
	{ K0411, K0412, K0413, K0414, _____, _____, K0415, K0416, K0417, _____ }, \
```

**Pull request closed without merging**